make-distribution.sh

```diff
@@ -188,6 +188,7 @@ echo "Build flags: $@" >> "$DISTDIR/RELEASE"
 # Copy jars
 cp "$SPARK_HOME"/assembly/target/scala*/*assembly*hadoop*.jar "$DISTDIR/lib/"
 cp "$SPARK_HOME"/examples/target/scala*/spark-examples*.jar "$DISTDIR/lib/"
+cp "$SPARK_HOME"/external/kafka/scala*/*kafka*assembly*.jar "$DISTDIR/lib/"
```

Comment:
Rather than packaging this with the release, can we just ask users to add the maven coordinates when launching it. This will add a fairly large amount to the binary size of Spark (especially if we add other ones in the future).